### ISSUE TITLE
Can parlMice continue imputation if insufficient convergence is detected?

### ISSUE THREAD

**Issue by benelot** [amices/mice#163]
Hi!

I am trying to setup parlMice in the following way:

```
mdf <- parlMice(...)
isNotConverged<-any(Rhats(mdf) > 1.1)
while (isNotConverged) {
    mdf<- parlMice(mdf,...)
   isNotConverged<-any(Rhats(mdf) > 1.1)
}
```

Is there any way this could be done? As far as I can see, I would have export the mice.mids interface.

```
imp1 <- mice(nhanes, maxit=1, seed = 123)
imp2 <- mice.mids(imp1)

# yields the same result as
imp <- mice(nhanes, maxit=2, seed = 123)

# verification
identical(imp$imp, imp2$imp)
```
from : https://rdrr.io/cran/mice/man/mice.mids.html

**Comment by gerkovink** [comment]
As a matter of fact; I am working on this. I'll keep you posted.

All the best,

Gerko

**Comment by benelot** [comment]
Exciting! Happy to hear any news!

**Comment by michaelwhammer** [comment]
Would these changes also allow `mice.mids` to be run in parallel as well? I often encounter scenarios where I want to run more iterations, but at the moment I would need to either run the additional iterations serially via `mice.mids` or simply rerun `parlmice` with more iterations.

**Comment by stefvanbuuren** [comment]
We are going to retire `parlmice()` in favour of `futuremice()` available in `mice 3.14.12`. 

Please reopen if this problem persists in `futuremice()`.
